docs/writing_how_do_i.rst

```diff
@@ -333,20 +333,31 @@ file.
 ----------------------------------------------------------------------------
 
 First, obtain the artifacts of the PR by adding this comment:
-``@BiocondaBot please fetch artifacts``. In the reply one finds the links to the
-built package and docker image. 
+``@BiocondaBot please fetch artifacts``. In the reply one finds the links a zip file containing
+the built package and docker image. Download this zip and extract it. For the following let
+``PACKAGES_DIR`` be the absolute path to the directory ``packages`` in the resulting unzipped directory
+and ``IMAGE_ZIP`` be the absolute path to the ``tar.gz`` file in the ``images`` directory in the unzipped directory.
 
 In order to test the tool with the package add the following to the planemo call::
 
-     $ planemo test ... --conda_channels LINK_TO_PACKAGE,conda-forge,bioconda,defaults ...
+     $ planemo test ... --conda_channels file://PACKAGES_DIR,conda-forge,bioconda,defaults ...
 
-For containerized testing the docker image needs to be loaded::
+For containerized testing we need to differentiate two cases:
 
-     $ curl  -L "LINK_TO_DOCKER_IMAGE.tar.gz" | gzip -dc | docker load
+1. the tool has a single requirement (that is fulfilled by the container)
+2. the tool has multiple requirements (in this case a docker image will be built on the fly using package)
 
-A planemo test will then simply use this image::
+For the former case the docker image that has been created by the bioconda CI needs to be loaded::
 
-     $ planemo test ... --biocontainers --no_conda_auto_init ...
+     $ gzip -dc IMAGE_ZIP | docker load
+
+and a planemo test can then simply use this image::
+
+     $ planemo test ... --biocontainers --no_dependency_resolution --no_conda_auto_init ...
+
+For the later case it suffices to call planemo as follows::
+
+     $ DEFAULT_MULLED_CONDA_CHANNELS="file://PACKAGES_DIR,conda-forge,bioconda" planemo test ... --biocontainers --no_dependency_resolution --no_conda_auto_init ...
```

Comment:
I would really like if this could be realized with `--conda_channels` (or a new CLI arg like `--mulled_conda_channels`), but I could not find out how.